ontology/geo.md

```diff
@@ -18,6 +18,7 @@ tracker: https://github.com/ufbmi/geographical-entity-ontology/issues
 activity_status: active
 repository: https://github.com/ufbmi/geographical-entity-ontology
 preferredPrefix: GEO
+domain: environment
```

Comment:
@hoganwr thoughts?

Reply:
Ok by me.  You can ignore my separate pull request for it, sorry.  Got confused.